From 80033361d9e65d56e997cc96d6e756ed8226e984 Mon Sep 17 00:00:00 2001
From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
Date: Tue, 8 Oct 2019 17:22:28 +0300
Subject: [PATCH] Add vspfilter config for h3ulcb-4x2g machine

This is just the copy from the original
gstvspfilter-h3ulcb_r8a7795.conf

Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 .../gstreamer1.0-plugin-vspfilter/gstvspfilter-h3ulcb-4x2g_r8a7795.conf | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-h3ulcb-4x2g_r8a7795.conf

diff --git a/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-h3ulcb-4x2g_r8a7795.conf b/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-h3ulcb-4x2g_r8a7795.conf
new file mode 100644
index 0000000..4807dc3
--- /dev/null
+++ b/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-h3ulcb-4x2g_r8a7795.conf
@@ -0,0 +1,2 @@
+input-device-name=/dev/video12
+output-device-name=/dev/video13
-- 
2.7.4

